### PR TITLE
MP-3-log-in-form

### PR DESCRIPTION
**Pull request by Dismemberment**
.

### CODE REVIEW

**Comment by DeylBlack** [comment]
seems like in this PR there are login only. so, in MP-3 we need create registration too. first commit is good and i'll wait next commit with registration template.

i think we can use ng-template and isLogin variables in .ts file

so, in html file you need create:
`<ng-container *ngIf="isLogin;else registration">
  <div class="log-in">
    <button click="isLogin = false">Registration</button>
  </div>
</ng-container>

<ng-template #registration>
  <div class="registration">
    <button click="isLogin = true">Log In</button>
  </div>
</ng-template>
`

And in ts file you need create 
`isLogin: boolean = true;`

File: src/app/log-in-page/log-in-page.component.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  width: 100%;
+  padding: 10px 0;
+  font-size: 15px;
+  color: white;
+  margin-bottom: 30px;
+  border: black;
+  border-bottom: 1px solid white;
+  outline: black;
+  background: transparent;
+}
```
Comment (Anastasiamatiek) [inline src/app/log-in-page/log-in-page.component.css:38]: this comment about code style. i think we need add empty string after all block style

File: src/app/log-in-page/log-in-page.component.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  color: white;
+  font-size: 15px;
+  text-decoration: none;
+  overflow: hidden;
+  transition: .5s;
+  margin-top: 10px;
+}
+
+.login-box a:hover {
+  background: white;
```
Comment (Anastasiamatiek) [inline src/app/log-in-page/log-in-page.component.css:71]: please use color code, ex: #ffffff

File: src/app/log-in-page/log-in-page.component.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  color: #000000;
+  border-radius: 5px;
+}
+
+.login-box a span {
+  position: absolute;
+  display: block;
+
```
Comment (Anastasiamatiek) [inline src/app/log-in-page/log-in-page.component.css:79]: remove empty string

File: src/app/log-in-page/log-in-page.component.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      <input type="text" name="" required="">
+      <label>Имя</label>
+    </div>
+    <div class="user-box">
+      <input type="password" name="" required="">
+      <label>Пароль</label>
+    </div>
+    <a href="">
+      <span></span>
+      Подтвердить
```
Comment (DeylBlack) [inline src/app/log-in-page/log-in-page.component.html:14]: i haven't seen it before. add text inside span tag

**Review by DeylBlack — APPROVED** [review]

**Pull request merged**